### DISCUSSION TITLE
Any good way to get geospatial indicis ?

### DISCUSSION THREAD

**Discussion by 1mike12** [Nozbe/WatermelonDB#1432] [Q&A]
From the docs, I see that theres a copy of the SQLite project forked that I'm assuming bundles a pre built binary. 

How would I go about adding SQLite modules to the SQLite bundled with this library? I'm considering something like the rtree module. https://www.sqlite.org/rtree.html

On the usage side, how would I specify a specific index type on the schema?

**Comment by Cheizr** [comment]
Hi, have you found a solution to store geo point data in watermelondb?
